v20.2/create-role.md

```diff
@@ -38,13 +38,259 @@ The `CREATE ROLE` [statement](sql-statements.html) creates SQL [roles](authoriza
+----------------------+---------------------------------------+------------
+admin                 |                                       | {}
+can_control_job       | CONTROLJOB, NOLOGIN                   | {}
+can_create_db         | CREATEDB, NOLOGIN                     | {}
+can_create_role       | CREATELOGIN, CREATEROLE, NOLOGIN      | {}
+can_login             | VALID UNTIL=2021-10-10 00:00:00+00:00 | {}
+can_manage_queries    | CANCELQUERY, NOLOGIN, VIEWACTIVITY    | {}
+no_options            | NOLOGIN                               | {}
+no_password           | NOLOGIN                               | {}
+root                  |                                       | {admin}
+(9 rows)
+~~~
+
+### Create a role that can control changefeeds
+
+~~~ sql
+root@:26257/defaultdb> CREATE ROLE can_control_changefeed WITH CONTROLCHANGEFEED;
+~~~
+
+~~~ sql
+root@:26257/defaultdb> SHOW ROLES;
+~~~
+
+~~~
+       username        |                options                | member_of
+-----------------------+---------------------------------------+------------
+admin                  |                                       | {}
+can_control_changefeed | CONTROLCHANGEFEED, NOLOGIN            | {}
+can_control_job        | CONTROLJOB, NOLOGIN                   | {}
+can_create_db          | CREATEDB, NOLOGIN                     | {}
+can_create_role        | CREATELOGIN, CREATEROLE, NOLOGIN      | {}
+can_login              | VALID UNTIL=2021-10-10 00:00:00+00:00 | {}
+can_manage_queries     | CANCELQUERY, NOLOGIN, VIEWACTIVITY    | {}
+no_options             | NOLOGIN                               | {}
+no_password            | NOLOGIN                               | {}
+root                   |                                       | {admin}
+(10 rows)
+~~~
+
+### Create a role that can modify cluster settings
+
+~~~ sql
+root@:26257/defaultdb> CREATE ROLE can_modify_cluster_setting WITH MODIFYCLUSTERSETTING;
+~~~
+
+~~~ sql
+root@:26257/defaultdb> SHOW ROLES;
+~~~
+
+~~~
+         username          |                options                | member_of
+---------------------------+---------------------------------------+------------
+admin                      |                                       | {}
+can_control_changefeed     | CONTROLCHANGEFEED, NOLOGIN            | {}
+can_control_job            | CONTROLJOB, NOLOGIN                   | {}
+can_create_db              | CREATEDB, NOLOGIN                     | {}
+can_create_role            | CREATELOGIN, CREATEROLE, NOLOGIN      | {}
+can_login                  | VALID UNTIL=2021-10-10 00:00:00+00:00 | {}
+can_manage_queries         | CANCELQUERY, NOLOGIN, VIEWACTIVITY    | {}
+can_modify_cluster_setting | MODIFYCLUSTERSETTING, NOLOGIN         | {}
+no_options                 | NOLOGIN                               | {}
+no_password                | NOLOGIN                               | {}
+root                       |                                       | {admin}
+(11 rows)
+~~~
+
+<!--
```

Comment:
Suggest deleting this commented out area - Git will keep it for us if we ever need it